packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/ui_components/phone_number_picker.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/widget_toolkit.dart';
+
+import '../../app_extensions.dart';
+import '../blocs/onboarding_phone_bloc.dart';
+import '../models/country_code.dart';
+import '../models/onboarding_phone_errors.dart';
+import '../services/phone_number_formatter.dart';
+import '../services/search_country_code_service.dart';
+
+class PhoneNumberPicker extends StatelessWidget {
+  const PhoneNumberPicker({
+    this.countryCodeTextStyle,
+    this.countryCodeBgColor = const Color(0xFFE1E1E1),
+    this.phoneNumberHint = 'XX XXX XXXX',
+    this.countryCodeHint = 'XXX',
+    super.key,
+  });
+
+  /// The text style of the country code text
+  final TextStyle? countryCodeTextStyle;
+
+  /// Background color of the country code segment
+  final Color countryCodeBgColor;
+
+  /// The hint text of the country code input
+  final String countryCodeHint;
+
+  /// The hint text of the phone number input
+  final String phoneNumberHint;
+
+  static const double _borderRadius = 16.0;
+
+  @override
+  Widget build(BuildContext context) => Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 8),
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                RxBlocBuilder<OnboardingPhoneBlocType, CountryCodeModel?>(
+                  state: (bloc) => bloc.states.countryCode,
+                  builder: (context, countryCodeSnapshot, bloc) =>
+                      _buildPhoneNumberPrefix(
+                    context,
+                    countryCodeSnapshot.data,
+                  ),
+                ),
+                RxBlocBuilder<OnboardingPhoneBlocType, String>(
+                  state: (bloc) => bloc.states.phoneNumber,
+                  builder: (context, numberSnapshot, bloc) => Expanded(
+                      child: _buildPhoneNumberInput(
```

Comment:
Can't we use the standart RxTextFormFieldBuilder ?

More information can be found here: https://medium.com/prime-holding-jsc/building-forms-in-flutter-454b8d65f48

Reply:
Fixed